indigo_app/forms.py

```diff
@@ -767,12 +779,14 @@ class WorkBulkActionsForm(forms.Form):
     save = forms.BooleanField()
     all_work_pks = forms.CharField(required=False)
     works = forms.ModelMultipleChoiceField(queryset=Work.objects, required=True)
+    user = forms.ModelChoiceField(queryset=User.objects)
```

Comment:
huh? just use request.user? otherwise I can impersonate you by submitting this with a different pk

Reply:
I guess we can just pass the user when we call .save_form() — I'll do that instead